cognite_toolkit/cognite_modules/examples/cdf_data_pipeline_files_valhall/functions/fu_context_files_oid_fileshare_annotation/handler.py

```diff
@@ -0,0 +1,833 @@
+import logging
+import os
+import re
+import time
+import traceback
+from datetime import datetime, timedelta
+from time import gmtime, strftime
+from typing import Any, Optional
+
+import yaml
+from cognite.client import ClientConfig, CogniteClient
+from cognite.client.credentials import OAuthClientCredentials
+from cognite.client.data_classes import (
+    Annotation,
+    AnnotationFilter,
+    ExtractionPipelineRun,
+    FileMetadata,
+    FileMetadataUpdate,
+)
+from cognite.client.exceptions import CogniteAPIError
+from cognite.logger import configure_logger
+
+# P&ID original file defaults
+orgMimeType = "application/pdf"
```

Comment:
This function probably deserves a rewrite. We recommend it to the user and if they starts run larger amounts of data through it I am afraid that they will hit the function time limit and thus have a poor experience.

Reply:
Agree - that this will not scale for a normal installation. So this type of code normally runs in f25e or customer infrastructure - here it's just an example not meant to be a production ready setup.